client-go/cmd/apps.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/apps"
+	"github.com/deis/deis/client-go/controller/models/config"
+)
+
+// AppCreate creates an app with the application
+func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
```

Comment:
There isn't a standard for this that I'm aware of in our codebase; up to you. Your argument makes sense to me.